catalog/transformer/catalog-transformer-pdf/src/main/java/ddf/catalog/transformer/input/pdf/PdfInputTransformer.java

```diff
@@ -219,59 +172,53 @@ private Metacard transformPdf(String id, PDDocument pdfDocument, String contentI
      * @param metacard    A mutable metacard to add the extracted data to
      */
     private void extractPdfMetadata(PDDocument pdfDocument, MetacardImpl metacard) {
-        StringBuilder metadataField = new StringBuilder("<pdf>");
-
-        PDMetadata documentCatalogMetadata = pdfDocument.getDocumentCatalog()
-                .getMetadata();
-        if (documentCatalogMetadata != null) {
-            try (InputStream inputStream = documentCatalogMetadata.createInputStream()) {
-                metadataField.append(IOUtils.toString(inputStream));
-            } catch (IOException e) {
-                LOGGER.warn("Couldn't read the PDF document catalog's metadata", e);
-            }
-        }
 
         PDDocumentInformation documentInformation = pdfDocument.getDocumentInformation();
 
         Calendar creationDate = documentInformation.getCreationDate();
         if (creationDate != null) {
             metacard.setCreatedDate(creationDate.getTime());
-            addXmlElement("creationDate", DATE_FORMAT.format(creationDate), metadataField);
+            metacard.setAttribute(Core.METACARD_CREATED, creationDate.getTime());
             LOGGER.debug("PDF Creation date was: {}", DATE_FORMAT.format(creationDate));
         }
 
         Calendar modificationDate = documentInformation.getModificationDate();
         if (modificationDate != null) {
             metacard.setModifiedDate(modificationDate.getTime());
-            addXmlElement("modificationDate", DATE_FORMAT.format(modificationDate), metadataField);
+            metacard.setAttribute(Core.METACARD_MODIFIED, modificationDate.getTime());
```

Comment:
metacard.modified shouldn't be set to the PDF modified date/time.  I would simply not set this attribute.